esy.lock/.gitattributes

```diff
@@ -0,0 +1,3 @@
+
+# Set eol to LF so files aren't converted to CRLF-eol on Windows.
+* text eol=lf linguist-generated
```

Comment:
@ospencer Oh, I just noticed that you accidentally ran `esy` in the top of the monorepo and committed an extra esy.lock directory here.